### DISCUSSION TITLE
In the published room, some assets that were displayed yesterday are no longer displayed

### DISCUSSION THREAD

**Discussion by ViGuselt** [mozilla/hubs#3841] [Spoke Questions]
**Description**
In the published room, when entering and walking there, all the assets Architecture Kit objects, Google Poly, and my 3D models created through Sketchfab disappeared. The scene editor has them. And yesterday they were also in the published scene. But today they all disappeared-only my uploaded Assets (photos and videos) remained.

**To Reproduce**
1. Go to 'https://hub.link/U2RTuRx'
2. Click on 'enter room'
3. Walk around the room. If there is no lighting, no walls behind the video, then there is an error.

**Expected behavior**
Behind all videos should be the wall assets from the Architecture Kit. There should be paths of stones (Assets Google Poly) leading from the spawn point in different directions, with a Spot Light placed next to them. Behind the drawings should be assets of easels. The scene editor (spoke) has it all (saved and published).

**Screenshots**
![image](https://user-images.githubusercontent.com/77488319/104717959-33485f00-573b-11eb-89ac-25978baf754b.png)
![image](https://user-images.githubusercontent.com/77488319/104718138-7a365480-573b-11eb-9a20-49f8695d3626.png)
![image](https://user-images.githubusercontent.com/77488319/104718155-80c4cc00-573b-11eb-80bc-8d113abfb7f6.png)
![image](https://user-images.githubusercontent.com/77488319/104718183-8c17f780-573b-11eb-9584-63bb9c2d54a6.png)
![image](https://user-images.githubusercontent.com/77488319/104718205-94703280-573b-11eb-9cd1-e9e38c5d63cd.png)

**Hardware** 
 - Device: Desktop
 - OS: Windows and Mac OS
 - Browser: Google Chrome, Microsoft Edge, Safari

**Additional context**

**Comment by j-conrad** [comment]
Without being able to edit the Spoke scene, I'm not exactly sure why those Architecture Kit pieces aren't showing. I am seeing a lot of errors in Firefox's console logs which indicate it may be related to memory.

As far as Google Poly assets go, that service was shut down by Google recently. (This bug is old enough that that probably was not the culprit.)
